### PR TITLE
Api improvements

### PR DESCRIPTION
**Pull request by aguspina**
Is this ok for you guys? @flbulgarelli @faloi

### CODE REVIEW

**Comment by aguspina** [comment]
Fixes #299

File: spec/controllers/api_controller_spec.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
       describe 'when not using filters' do
         before { get :index }
-        it { expect(response.body).to eq '{"guides":[{"id":1,"github_repository":"flbulgarelli/mumuki-sample-exercises","name":"guide_1","language_id":1,"path_id":null,"position":null}]}' }
```
Comment (flbulgarelli) [inline spec/controllers/api_controller_spec.rb]: Could you add a test for the scenario when we actually have exercises in the guide?

**Comment by flbulgarelli** [comment]
@aguspina Looks good, just add the missing test an proceed with merge

**Comment by faloi** [comment]
:+1:

File: spec/controllers/api_controller_spec.rb (hunk 1 above)
Comment (aguspina) [inline spec/controllers/api_controller_spec.rb]: Done in 9a3e2079eec5c986a2ec1860d0ec2747ef7257e3

**Pull request merged**